api/src/main/scala/dx/api/DxApi.scala

```diff
@@ -189,6 +191,7 @@ case class DxApi(version: String = "1.0.0", dxEnv: DXEnvironment = DxApi.default
       case job: DxJob           => callObject(DXAPI.jobAddTags[JsonNode], job.id, fields)
       case project: DxProject   => callObject(DXAPI.projectAddTags[JsonNode], project.id, fields)
       case record: DxRecord     => callObject(DXAPI.recordAddTags[JsonNode], record.id, fields)
+      case database: DxDatabase => callObject(DXAPI.databaseAddTags[JsonNode], database.id, fields)
```

Comment:
Just a question: so dbcluster does not support addTags? Also only the file object has removeTags method but it (as well as rename, addTags and setProperties)should be working for all DxDataObjects 
https://documentation.dnanexus.com/developer/api/introduction-to-data-object-metadata/tags

Reply:
Thanks for noticing! I missed that DBcluster is also a data object. I'll add the missing methods.